src/Microsoft.PowerShell.Commands.Utility/commands/utility/convert-HTML.cs

```diff
@@ -228,6 +228,56 @@ public string[] PreContent
         private string[] _preContent;
 
         /// <summary>
+        /// Sets and Gets the meta property of the HTML head
+        /// </summary>
+        /// <returns></returns>
+        [Parameter(ParameterSetName = "Page")]
+        [ValidateNotNullOrEmpty]
+        public Hashtable Meta
+        {
+            get
+            {
+                return _meta;
+            }
+            set
+            {
+                _meta = value;
+                _metaSpecified = true;
+            }
+        }
+        private Hashtable _meta;
+        private bool _metaSpecified = false;
+
+        /// <summary>
+        /// Specifies the charset encoding for the HTML document
+        /// </summary>
+        [Parameter(ParameterSetName = "Page")]
+        [ValidateNotNullOrEmpty]
+        public string Charset
```

Comment:
Can we add a [ValidatePatternAttribute](https://msdn.microsoft.com/en-us/library/ms714454%28v=vs.85%29.aspx?f=255&MSPPError=-2147217396) and verify that the parameter is a valid Charset.  For example, charset should not have a `"`, `<` or `>` in the parameter.